packages/core/contracts/cross-chain-oracle/chain-adapters/Optimism_ChildMessenger.sol

```diff
@@ -91,10 +100,10 @@ contract Optimism_ChildMessenger is OVM_CrossDomainEnabled, ChildMessengerInterf
      */
     function processMessageFromCrossChainParent(bytes memory data, address target)
         public
-        override
```

Comment:
why is this no longer override? do we not want this to be the standard interface for inbound messages from the parent -> child?

Reply:
same question on the parent in the following file.

Reply:
I don't think this is standard for all networks. Only Optimism so far, and Arbitrum will work the same. But Polygon works differently so I don't think it should go in the interface